dom-event-delegation/dom-event-delegation-quiz-notes.md

```diff
@@ -0,0 +1,50 @@
+# dom-event-delegation-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the `event.target`?
+  the event.target targets the html element that we are applying our addEventListener to
```

Comment:
Not quite! What's the difference between `event.target` & `event.currentTarget`?

Reply:
Ohhh! so the event.target is the event that triggered the event but the event.currentTarget is the target that the eventListener is added to. 

So in this case, the event.target is either the button, the span, or the li. However, our event.currentTarget is the element with the task-list class.